app/controllers/users/invitation_resendings_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Users::InvitationResendingsController < ApplicationController
```

Comment:
Minor: I think `InvitationResendController` is a littler easier to read, but I don't feel strongly about it -  I appreciate it's a bit of an invasive change to make.

Reply:
I dithered over this for ages, but you've convinced me. It'll be quite a faff, but I'll change it to `InvitationResendsController` (note the "Resend" will need to be plural).

Reply:
Addressed in https://github.com/alphagov/signon/pull/2559/commits/e23e19a4d353139f90d7e22e9dd9d49e9e7470bf.